### PR TITLE
Fix #9794: Malformed .gz inventory breaks upload process

### PR DESCRIPTION
**Pull request by armeniaca**

### CODE REVIEW

File: tools/send-clean.sh [hunk 1]
```diff
@@ -67,6 +67,16 @@ fi
 if [ "${BASENAME##*.}" = "gz" ]
 then
   ${GZIP_BINARY} --force --quiet --decompress ${FILENAME}
+  UNZIP_COMMAND_RET=$?
+  if [ $UNZIP_COMMAND_RET -ne 0 ]
+  then
+    echo "ERROR: failed to decompress inventory in ${FILENAME} az gzip, putting it in the failed directory"
+    mv "${FILENAME}" "${FAILEDDIR}/`basename ${FILENAME}`-$(date --rfc-3339=date)"
```
Comment (peckpeck) [inline tools/send-clean.sh]: why do you mix `` and $() ?

File: tools/send-clean.sh [hunk 2]
```diff
@@ -67,6 +67,16 @@ fi
...
+    mv "${FILENAME_SIGN}" "${FAILEDDIR}/`basename ${FILENAME_SIGN}`-$(date --rfc-3339=date)"
```
Comment (peckpeck) [inline tools/send-clean.sh]: There is not always a signature file, you should avoid errors here with a [ -f ] &&

Comment (armeniaca) [inline tools/send-clean.sh]: same as above. will rework that part of code without using your logic, but as I see well suited.

**Comment by peckpeck** [comment]
Sorry for the original mixed style and thank you for the PR.

**Pull request merged**